src/piprules/namespace_pkgs.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2020 128 Technology, Inc.
+# Copyright 2016 Dillon Giacoppo github.com/dillon-giacoppo
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The contents of this file were mostly copied from:
+# https://github.com/dillon-giacoppo/rules_python_external
+# specifically the files namespace_pkgs.py and wheel.py.
+# 128 Technology has made changes to these files and combined them into this file.
```

Comment:
could you give a TLDR of what you had to modify?

I'm trying to flip between this and https://github.com/dillon-giacoppo/rules_python_external/blob/master/extract_wheels/lib/namespace_pkgs.py and can't spot much

Reply:
From what I can see the main changes were removing type hints and bringing in `_get_dist_info` logic from `wheel.py`, plus minor formatting + style changes (like prefixing function names with `_`)